templates/partials/list-content.html

```diff
@@ -0,0 +1,19 @@
+
+{% set title = content.title %}
+{% set contents = content.contents %}
+
+{% call onsQuestion({
+  "id": question.id,
+  "title": title,
```

Comment:
Why do we use `onsQuestion`?. question won't exist, hence the id will be not be defined.

Reply:
No need, removed.